features/argument_transforms/whitespace.feature

```diff
@@ -10,6 +10,13 @@ Feature: Whitespace
       """
     Then the value of the response body is equal to `<expected>`
 
+  Examples:
+    | input              | expected     |
```

Comment:
the content below intentionally includes tabs so this isn't going to line up consistently